psi4/src/psi4/scfgrad/jk_grad.cc

```diff
@@ -115,7 +115,7 @@ void JKGrad::common_init()
     omp_num_threads_ = Process::environment.get_n_threads();
 #endif
 
-    cutoff_ = 0.0;
+    cutoff_ = 1E-12;
```

Comment:
This will only change the printing (and perhaps incorrectly). The JK cutoff is set above, I believe this should be:
```
cutoff_ = Process::environment.options.get_double("INTS_TOLERANCE")
```

Reply:
Minor reminder here.

Reply:
Gentle reminder here.

Reply:
@dgasmith thanks for the pings, looks like I missed the first one. Pointing fingers is more effective ;)